paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -141,6 +145,74 @@ def sync_secrets(
     if not os.path.isdir(secret_dir):
         log.debug(f"No secrets dir for {service}")
         return True
+
+    # Update boto key secrets
+    service_dir = os.path.join(soa_dir, service)
+    kubernetes_config_file = f"kubernetes-{cluster}.yaml"
```

Comment:
nit: presumably there are abstractions elsewhere for getting a reference to the cluster yaml config file?

Reply:
I think that extends to some of the below too. We have abstractions for reading the yelpsoa configs and properties within. https://github.com/Yelp/paasta/blob/master/paasta_tools/kubernetes_tools.py#L444 is probably where you want to start.